### PR TITLE
Add AuthenticationX

### PR DESCRIPTION
**Pull request by yezz123**
@JefferyHus @yjose i just add my last package and fix some of them 🚀

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -218,24 +218,20 @@
 - [Laravel, Vue & Admin LTE Boilerplate](https://github.com/hbakouane/adminlte-laravel-vue) - Laravel, Vue and Admin LTE integrated in one project.
 - [PHPFlasher](https://github.com/php-flasher/php-flasher) - A powerful and flexible flash notification system for PHP
 With Symfony and Laravel implementations. by [@yoeunes](https://github.com/yoeunes)
- 
+
 #### Python
 
 - [sourcerer-io-scrapper](https://github.com/Stormix/sourcerer-io-scrapper) - Because they don't have an API. <https://sourcerer.io/> by [@Stormix](https://github.com/Stormix)
-- [ping360_sonar](https://github.com/CentraleNantesRobotics/ping360_sonar) - ROS package for Blue Robotics Ping360 Sonar by [@Stormix](https://github.com/Stormix)
+- [ping360-sonar](https://github.com/CentraleNantesRobotics/ping360_sonar) - ROS package for Blue Robotics Ping360 Sonar by [@Stormix](https://github.com/Stormix)
```
Comment (JefferyHus) [inline readme.md:225]: 🤔 , 2 major questions:

- Is this 100% a Moroccan organization? If not, what makes it a Moroccan contribution?
- The package name is actually with a `_` is there a reason to use the dash?

**Review by JefferyHus — COMMENTED** [review]
LGTM !! multiple positioning makes it hard to review 😅

File: readme.md (hunk 1 above)
Comment (JefferyHus) [reply readme.md:225]: There is also the fact that some old commits were pushed as well 🤔

Comment (yezz123) [reply readme.md:225]: let's ping the creator of this one 😅

@Stormix is this one still active?

Comment (Stormix) [reply readme.md:225]: I wrote most of the code on that repo and the org only serves as a fancy nameplate. I would define a Moroccan contribution as any contribution made by an author with Moroccan national. And yes, the repo is still being maintained.

Comment (Stormix) [reply readme.md:225]: @yezz123 @JefferyHus I left a reply above, but still wonder why is this issue being raised in a PR for an other contribution 🤔 and not the original PR.

**Comment by yezz123** [comment]
Yo @JefferyHus if this PR clean we could merge it

**Pull request merged**